src/app/OperationalSessionSetup.cpp

```diff
@@ -262,8 +187,97 @@ CHIP_ERROR OperationalSessionSetup::EstablishConnection(const ReliableMessagePro
     return CHIP_NO_ERROR;
 }
 
+void OperationalSessionSetup::Connect(Callback::Callback<OnDeviceConnected> * onConnection,
+                                      Callback::Callback<OnDeviceConnectionFailure> * onFailure,
+                                      Callback::Callback<OnSetupFailure> * onSetupFailure)
```

Comment:
For future reference: This would have been a lot easier to review if this were at the old location of this code in the file.  Might be worth moving it before this PR merges, to make diffs more readable for future people reading the history.